main_page.py

```diff
@@ -0,0 +1,71 @@
+from selenium.webdriver.remote.webdriver import WebDriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as ec
+import my_account_page
+
+
+class Constant_newsletter:
+    newsletter_name = 'adsdada'
+    newsletter_mail = "assd@gmail.com"
+    input_name_box = (By.ID, 'es_txt_name')
+    input_mail_box = (By.ID, 'es_txt_email')
+    subscribe_button = (By.ID, 'es_txt_button')
+    confirmation_text_selector = (By.ID, 'es_msg')
+    confirmation_text = 'Successfully Subscribed'
+
+class Constant_user_abcd:
+    haslo = "Abcd123456789!"
+    uzytkownik = "abcdf@gmail.com"
+
+
+class Constant_user_automaty:
+    login = 'l.miotk.py'
+    password = "Aut0m@ty$a$uper!"
+
+
+class Cosntant_most_wnated:
+    zakladka_most_wanted = (By.XPATH, '//*[@id="menu-item-128"]/a')
+    naglowek_most_wanted = (By.XPATH, '//*[@id="page"]/div/div/div[2]/div/h1')
+    most_wanted_txt = "most wanted"
+
+
+class Constant_Buy_product:
```

Comment:
```suggestion
class ConstantBuyProduct:
```
https://peps.python.org/pep-0008/#class-names